sql/testing/path_test.cc

```diff
@@ -138,13 +130,23 @@ BOOST_AUTO_TEST_CASE(test_coords_parsing)
     }
 
     {
+        Path coords1 = Path::parse("..");
+        BOOST_CHECK_EQUAL(coords1.size(), 3);
+        BOOST_CHECK_EQUAL(coords1.toUtf8String(), "..");
```

Comment:
The "" aren't actually necessary as it's not ambiguous.  Do you think we should put them in anyway?